app/controllers/camaleon_cms/admin/posts_controller.rb

```diff
@@ -53,7 +53,13 @@ def index
     r = {posts: @posts, post_type: @post_type, btns: @btns, all_posts: posts_all, render: 'index', per_page: per_page }
     hooks_run("list_post", r)
     add_breadcrumb "#{@btns[params[:s].to_sym]}" if params[:s].present?
-    @posts = r[:posts].paginate(:page => params[:page], :per_page => r[:per_page])
+
+    # Return latest posts for news
+    if params[:post_type_id] == '23'
```

Comment:
I wonder if this version works:

@posts = r[:posts].paginate(:page => params[:page], :per_page => r[:per_page])

\# Return latest posts for news
if params[:post_type_id] == '23'
  @posts = @posts.latest  
end